modules/utxo-core/package.json

```diff
@@ -20,14 +20,14 @@
     "./testutil/descriptor": "./dist/src/testutil/descriptor/index.js"
   },
   "scripts": {
-    "build": "yarn tsc --build --incremental --verbose .",
+    "build": "yarn tsc --build --incremental --verbose . && cpx \"test/descriptor/psbt/fixtures/**/*\" dist/test/descriptor/psbt/fixtures",
```

Comment:
This will cause our deployed bundle to include these fixture files, which doesn't seem right